docs/ExternalResults.asciidoc

```diff
@@ -0,0 +1,130 @@
+
+[[installing]]
+= openQA test harness result processing
+:toc: left
+:toclevels: 6
+:author: openQA developers
+
+== Introduction
+:parser-formats: <<parser-formats,parsers>>
+:api-endpoint: <<webapi-endpoint,Web Api endpoint>>
+
+From time to time, a test developer might want to use openQA to execute a test
+suite from a different test harness than openQA, but still use openQA to setup test
+scenarios and prepare the environment for a test suite run; for this case openQA has
+the hability to process logs from external harnesses, and display the results integrated
+within the job results of the webUI.
+
+One could say that a Test Harness is supported if its output is compatible with
+the available {parser-format}, such as LTP, and also xUnit or JUnit, but this can
+be easily extended to include more formats, such as RSpec or TAP.
+
+The requirements to use this functionality, are quite simple:
+
+* The test harness must produce a compatible format with supported {parser-format}.
+* The test results can be uploaded via +testapi::parse_extra_log+ within an openQA tests.
+* The test results can also be uploaded via web {api-endpoint}.
+
+openQA will store these results in its own internal format for easier presentation,
+but still will allow the original file to be downloaded.
+
+== Usage
```

Comment:
@richiejp @metan Is this information enough for you guys? on how to use the parser and stuff?

Reply:
Yeah, so calling `parse_extra_log` will also make the file available in the test 'logs & assets' tab? This will work great for us. The only problem is possibly with how you are saying the results will be displayed.